### PR TITLE
translated a few stat descriptions

### PR DESCRIPTION
**Pull request by normalpanchan**
will do more later

### CODE REVIEW

**Comment by Kyle873** [comment]
@Pan will need you to add a "\n" line break between Defense and stats on lines 174, 179, 184, 189, and 194.

File: Affix Descriptions.json [hunk 1]
```diff
@@ -496,78 +496,78 @@
...
-    "Text": "",
-    "Enabled": false
+    "Text": "Boosts ice elemental damage by 60%",
+    "Enabled": true
   },
   "820109": {
     "OriginalText": "雷属性の攻撃によるダメージが\n２０％軽滅される。",
-    "Text": "",
-    "Enabled": false
+    "Text": "Boosts thunder elemental damage by 20%",
```
Comment (Kyle873) [inline Affix Descriptions.json]: thunder -> lightning for consistency

**Comment by Kyle873** [comment]
Periods should be put at the ends to make them sentences, to match the JP text.
Elements such as Fire, Ice, Lightning etc should be capatalized.

**Comment by SynthSy** [comment]
Merging, but do note to keep the original ※. This is known as a ricemark, and it's used for annotations (as you have noticed.)
Good work though!

**Pull request merged**